Makefile

```diff
@@ -1,5 +1,5 @@
 PD_PKG := github.com/pingcap/pd
-VENDOR := $(shell rm -f _vendor/src/$(PD_PKG) &&\
+VENDOR := $(shell rm -rf _vendor/src/$(PD_PKG) &&\
```

Comment:
Here is not linking the vendor path but link the pd root path to _vendor. Please refer #679 for more details.